configure.ac

```diff
@@ -1,6 +1,6 @@
 AC_PREREQ(2.63)
 AC_INIT([libvigor], [m4_esyscmd([./version.sh])], [bugs@niftylogic.com])
-AX_ABI_VERSION([2], [0], [1])
+AX_ABI_VERSION([2], [1], [1])
```

Comment:
I'm not sure this version bump is correct.  Given that the structures are changing, might we run into a problem where compiled binaries are referencing 2.x.x ABI, but passing a hash_t structure (statically allocated in the image) that is 8-32 bytes short?